src/test/java/de/tum/in/www1/artemis/exercise/programmingexercise/ProgrammingExerciseResultTestService.java

```diff
@@ -98,6 +98,8 @@ public class ProgrammingExerciseResultTestService {
     @Autowired
     private ParticipationUtilService participationUtilService;
 
+    private Course course;
```

Comment:
Codacy: "Perhaps 'course' could be replaced by a local variable."
What do you think?

Reply:
I could be, but other tests might also need the course object in the future, so I'd like to keep it here.